### ISSUE TITLE
Utterance terminology

### ISSUE THREAD

**Issue by zepumph** [phetsims/utterance-queue#27]
I have noticed a lot of indiscretion as it pertains to the terminology involved with Utterances. I would like to potentially set it straight, or at least get on the same page. Here are the words that I come in contact with quite often, and what I would like the definitions to be:

* Alert (_v_): to release an Utterance to aria-live, and solely a description term. Used to describe what comes from description aria-live implementation.
* Announce (_v_): verbage to describe the UtteranceQueue "releasing" an utterance to the announcer.
* Speak: A voicing-specific word. I think we have already hit this one pretty hard conventionally in the Voicing.js mixin
* Alert (_n_): the element that you get through Aria live, only for description.
* Response (_n_): the element that you get through voicingManager speaking, only for voicing.

Ways in which the above differs from terminology in master, and potential ways this could adhere to the above if we want to adopt that:
* `Utterance.alert`. This could be renamed to `Utterance.announcement`
* Currently there is a fair bit of understanding that a "response" could be description of voicing. As a noun, I really like using this for Voicing, but for description we could call it something like "object response alert". This is a bit weird though.
* Lots of documentation. In general I think we just use whatever feels best at the time. We should be consistent if we feel like it is worth solidifying this terminology.

@jessegreenberg the first question is do you think any work on this issue is worth while, even minor steps? The next question is about reviewing my proposal and going from there. Marking for meeting, but assigning for comment if async comes first.

Thanks again for the marathon meeting this afternoon. It really helped progress the work in UtteranceQueue!

**Comment by zepumph** [comment]
@jessegreenberg and I discussed today, and we note that there is discrepancy in how we use this terminology. That said we want to talk about it more next week after thinking about it for a week.

> * `Utterance.alert`. This could be renamed to `Utterance.announcement`

We are pretty sure that this would be an improvement.

See you next week!

**Comment by zepumph** [comment]
What about `AlertManager`? Does that mean it only can support description Utterances? (hopefully not!)

**Comment by zepumph** [comment]
Tagging https://github.com/phetsims/utterance-queue/issues/13
